LoRaEngine/Directory.Build.props

```diff
@@ -0,0 +1,30 @@
+<Project>
+    <ItemGroup>
+        <PackageReference Include="Microsoft.CodeAnalysis.FxCopAnalyzers" Version="3.3.2">
```

Comment:
Since we are on the .NET 5 SDK since merge of PR #437, [the analyzers are already included](https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/overview) and can be enabled via the following properties:

```xml
        <EnableNETAnalyzers>true</EnableNETAnalyzers>
        <EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
        <AnalysisMode>AllEnabledByDefault</AnalysisMode>
        <AnalysisLevel>latest</AnalysisLevel>
```